docs/grpc/docs.md

```diff
@@ -1432,6 +1432,8 @@ Note: 1kB = 1 vector of size 256. |
 | search_max_oversampling | [float](#float) | optional |  |
 | upsert_max_batchsize | [uint64](#uint64) | optional |  |
 | max_collection_vector_size_bytes | [uint64](#uint64) | optional |  |
+| read_rate_limit_per_sec | [uint32](#uint32) | optional |  |
+| write_rate_limit_per_sec | [uint32](#uint32) | optional |  |
```

Comment:
rate limit per second won't allow us to configure bursts. Could we switch to requests per minute?

Reply:
Internally the rate limiter computes the number of requests allowed per second.

```rust
let tokens_per_sec = rate.requests_num() as f64 / rate.period().as_secs_f64();
```

Having an input of `10 req/s` is equivalent to `600 req/m`.

Those tokens are replenished in real time based on the elapsed time since the last check.
We do start at full capacity to allow burst on start.

Here is the impl:

```rust
pub fn check(&mut self) -> bool {
        let now = Instant::now();
        let elapsed = now.duration_since(self.last_check);
        self.last_check = now;

        // Refill tokens based on elapsed time.
        self.tokens += self.tokens_per_sec * elapsed.as_secs_f64();
        if self.tokens > self.capacity as f64 {
            self.tokens = self.capacity as f64;
        }

        if self.tokens >= 1.0 {
            self.tokens -= 1.0; // Consume one token.
            true // Request allowed.
        } else {
            false // Request denied.
        }
    }
```

Which kind of burst scenario are you concerned about?

Reply:
Discussed with Andrey regarding his concerns, he wants to be able to have an actual burst capacity of one minute worth of tokens and not one second.

I will merge this PR as it is because it is tidy and has been reviewed.
Then propose the adjusted implementation in a new PR for clarity.